app/plugin_api.go

```diff
@@ -98,10 +98,22 @@ func (api *PluginAPI) SavePluginConfig(pluginConfig map[string]interface{}) *mod
 	return api.app.SaveConfig(cfg, true)
 }
 
+func (api *PluginAPI) GetLicense() *model.License {
+	return api.app.License()
+}
+
 func (api *PluginAPI) GetServerVersion() string {
 	return model.CurrentVersion
 }
 
+func (api *PluginAPI) GetSystemInstallDate() (int64, *model.AppError) {
```

Comment:
Out of interest: What is the use case for this?

Reply:
For both of these, they're to give some context to the Net Promoter Score survey results, so we can see if TE vs EE or how old the server is have an effect on how much users like Mattermost.

I don't know if the spec for the feature is public, but we're gathering [these fields](https://github.com/hmhealey/mattermost-plugin-nps/blob/master/server/segment.go#L54) with user feedback

Reply:
Makes sense, thanks for the explanation!